### ISSUE TITLE
Ready-made assemblies

### ISSUE THREAD

**Issue by JekSun97** [MizunagiKB/gd_cubism#60]
Could you create ready-made dll libraries, the library is very difficult to compile :(

**Comment by MizunagiKB** [comment]
As you rightly pointed out, it would indeed be very convenient if we could download the pre-built binaries from the Godot Asset Library.

I completely agree with you.
However, GDCubism is allowed to be publicly released only by adhering to the licensing terms set by Live2D Inc. Therefore, at this point, we are unable to adopt a method of distributing pre-built binaries.

If you are experiencing difficulties building GDCubism, depending on the specifics of the issue, we may be able to offer some suggestions.

**Comment by JekSun97** [comment]
It's a pity :(

My procedure:
1. I downloaded godot-cpp-4.1 and put all the files and folders in godot-cpp folder, then I used Scons to build it
2. I downloaded and installed CubismNativeFramework from here (https://github.com/Live2D/CubismNativeFramework/releases/tag/5-r.1-beta.3), and put the folder "CubismNativeFramework-5-r.1-beta .3" to the "thirdparty" folder. It was not possible to assemble it, if it is necessary, it does not see the Live2DCubismCore.h file
3. I enter the Scons command to build the extension: scons platform=windows arch=x86_64 target=template_release

I end up getting this error:
![1](https://github.com/MizunagiKB/gd_cubism/assets/130399274/2b093399-771d-450c-9154-970d010181b7)

**Comment by JekSun97** [comment]
I believe the problem is that I haven't put together the CubismNativeFramework, could you make a YouTube video on how to do it correctly, or write instructions if you don't mind?

**Comment by MizunagiKB** [comment]
Thank you for your explanation of the procedure.
It appears that the document I created may have led to incorrect steps being taken.

Please obtain the library that `GDCubism` depends on from the official Live2D site, not GitHub.

You can download the Cubism 5 SDK for Native R1 beta4 from the following location (as of February 17, 2024)
https://www.live2d.com/en/sdk/download/native/

If you unzip it in the _thirdparty_ folder, you should be able to build `GDCubism` in your environment.

> I have corrected the document in the 0.5 branch.
> https://github.com/MizunagiKB/gd_cubism/blob/0.5/docs/BUILD.en.adoc

**Comment by MizunagiKB** [comment]
This issue appears to have been resolved, so I will be closing it

**Comment by JekSun97** [comment]
@MizunagiKB All the BUILD.md for describing the assembly have disappeared somewhere

**Comment by MizunagiKB** [comment]
Starting from version 0.6, we have made changes to generate AsciiDoc with a search function to make the document easier to read.

The build can be found at the following location:

https://mizunagikb.github.io/gd_cubism/gd_cubism/0.6/en/build.html

We forgot to update the link from the README.adoc, README.en.adoc on GitHub. The links are broken not only for the build but also for other sections. A correction is needed, so we have reopened the ticket.
